src/handlers/AllHandler.ts

```diff
@@ -22,7 +22,7 @@ export default class AllHandler extends GroupMeHandler {
    * it is given.
    * @param groupMeInfo
    */
-  shouldHandle = (groupMeInfo: GroupMeInfo): boolean => {
+  shouldHandle = (groupMeInfo: Readonly<GroupMeInfo>): boolean => {
```

Comment:
worth Having GroupMeInfo tydef as Readonly<_GroupMeInfo>?

Reply:
nah you will find that eventually it gets really annoying if GroupMeInfo was readonly itself. Otherwise you are left using spread operators and creating new objects for generally no reason really.

Reply:
I don't (_currently_) see a reason why we should ever mutate `GroupMeInfo` but sounds good to me :)